start-everything.ps1

```diff
@@ -31,4 +31,5 @@ Write-Host ("Starting Remixd - local remix directories access point") -Foregroun
 $dir = (Get-ScriptDirectory) + "/solidity"
 Write-Host ("Remixd shared directory: " + ($dir))  -ForegroundColor blue
 Write-Host ("TO CLOSE - JUST CTRL+C") -ForegroundColor red -BackgroundColor white
-remixd -s $dir --remix-ide http://localhost:8080/ 

```

Comment:
This program is important, it allows  You to use current directory for editing in the redix editor. We can open the url earlier and then start the daemon for the drive usage.